src/LaminasViewRenderer.php

```diff
@@ -145,7 +145,7 @@ public function getPaths(): array
         $paths = [];
 
         /**
-         * @var array<string, SplStack<string>> $pathStack
+         * @var array<array-key, SplStack<string>> $pathStack
```

Comment:
`as $namespace` seems to clash with `array-key` here :thinking: 

I don't think this is correct.

Reply:
This type was previously, incorrectly forced by me I think. There's a check for `is_int` on the key which psalm 5 rightly complains about because of the forced type - effectively, `$pathStack` is `array<namespace|int, SplStack<string>>` as far as I can tell.